### PR TITLE
Support Hash type for attrs

### PR DESCRIPTION
**Pull request by bforma**

### CODE REVIEW

**Comment by lvonk** [comment]
Nice, but I don't this will work with a Hash containing ValueObjects... can you check? Because that might be confusing.

**Comment by bforma** [comment]
> Nice, but I don't this will work with a Hash containing ValueObjects... can you check? Because that might be confusing.

Sure, I'll add some specs with 'complex' Hash structures.

**Comment by lvonk** [comment]
ping @bforma

**Comment by lvonk** [comment]
Closing due to inactivity

File: spec/lib/sequent/core/helpers/type_conversion_support_spec.rb [hunk 1]
```diff
@@ -264,6 +264,47 @@ class CommandWithDateTime < Sequent::Core::BaseCommand
 
   end
 
+  context Hash do
+    class CommandWithHash < Sequent::Core::BaseCommand
+      attrs value: Hash
```
Comment (lvonk) [inline spec/lib/sequent/core/helpers/type_conversion_support_spec.rb:269]: I don't think a Hash containing a ValueObject works. Can you add a test for this? I think if it does not work we should fail fast, and document that it is not supported.

**Review by lvonk — COMMENTED** [review]
Can you check if a Hash containing a ValueObject works?

File: spec/lib/sequent/core/helpers/type_conversion_support_spec.rb (hunk 1 above)
Comment (bforma) [reply spec/lib/sequent/core/helpers/type_conversion_support_spec.rb:269]: Yes I was going to fix this.

**Pull request closed without merging**